solidity/contracts/token/HypERC20Collateral.sol

```diff
@@ -63,6 +65,20 @@ contract HypERC20Collateral is FungibleTokenRouter, MovableCollateralRouter {
         return wrappedToken.balanceOf(_account);
     }
 
+    function quoteTransferRemote(
+        uint32 _destinationDomain,
+        bytes32 _recipient,
+        uint256 _amount
+    ) external view virtual override returns (Quote[] memory quotes) {
+        quotes = new Quote[](2);
+        quotes[0] = _quoteTransferRemote(
+            _destinationDomain,
+            _recipient,
+            _amount
+        )[0];
+        quotes[1] = Quote({token: address(wrappedToken), amount: _amount});
```

Comment:
Is this intentional? I guess this is another "nam didn't specify requirements", but imo the quote should only be for any other amounts other than the actual tokens that are being transferred